docs/source/demo.rst

```diff
@@ -7,41 +7,63 @@ Demo
 Running a demo site
```

Comment:
If we want to hold on to the Docker and/or Heroku demo alternatives, it's probably a good idea to change the title to "Running a demo site with Cookiecutter"

Reply:
I thought we were dropping them, or at least the Heroku one? Anyway, not sure if that's a part of this PR. I'd prefer using this PR to get the demo running as local python again.

Reply:
Ok with changing the title here, but I'd handle the decision to drop the rest of the docs somewhere else.

Reply:
I prefer to drop the Heroku documentation/support but I agree that is should be handled in other issue. We will try to keep the Docker support so changing the title to "Running a demo site with Cookiecutter" will be useful